src/main/kotlin/com/hibob/academy/employeeFeedback/dao/FeedbackResponseDao.kt

```diff
@@ -0,0 +1,73 @@
+package com.hibob.academy.employeeFeedback.dao
+
+import com.hibob.academy.employeeFeedback.dao.table.FeedbackResponseTable
+import com.hibob.academy.employeeFeedback.model.CreateResponseRequestWithFeedbackId
+import com.hibob.academy.employeeFeedback.model.LoggedInUser
+import com.hibob.academy.employeeFeedback.model.Response
+import com.hibob.academy.employeeFeedback.model.UpdateFeedbackResponseRequest
+import jakarta.inject.Inject
+import jakarta.ws.rs.BadRequestException
+import org.jooq.DSLContext
+import org.jooq.RecordMapper
+import org.jooq.Record
+import org.jooq.impl.DSL
+import org.springframework.stereotype.Repository
+
+@Repository
+class FeedbackResponseDao @Inject constructor(
+    private val sql: DSLContext
+) {
+
+    private val feedbackResponseTable = FeedbackResponseTable.instance
+
+    private val feedbackResponseMapper = RecordMapper<Record, Response>
+    { record ->
+        Response(
+            id = record[feedbackResponseTable.id],
+            feedbackId = record[feedbackResponseTable.feedbackId],
+            responderId = record[feedbackResponseTable.responderId],
+            text = record[feedbackResponseTable.text],
+            createdAt = record[feedbackResponseTable.createdAt].toLocalDate(),
+        )
+    }
+
+    fun create(createRespondRequest: CreateResponseRequestWithFeedbackId, loggedInUser: LoggedInUser) : Long {
```

Comment:
it makes sense to put the variables in a function by their importance, so let's keep sending loggedInUser or companyId as a first parameter in functions.